recipes/pytweening/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.0.3" %}
+{% set name = "PyTweening" %}
+
+package:
+  name: pytweening
+  version: {{ version }}
+
+source:
+  fn: PyTweening-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 4b608a570f4dccf2201e898f643c2a12372eb1d71a3dbc7e778771b603ca248b
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - pytweening
+
+about:
+  home: https://github.com/asweigart/pytweening
+  license: BSD 3-Clause
+  license_family: BSD
```

Comment:
`license_file`: 

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
ok see https://github.com/asweigart/pytweening/pull/3